testdata/p4_16_samples/list3.p4

```diff
@@ -1,6 +1,6 @@
 #include <core.p4>
 
-typedef int EntryPriority_t;
+typedef int<8> EntryPriority_t;
```

Comment:
why modify this test?
These are all regression tests, if you modify them they no longer serve their purpose.
You should add new tests if you want to check new properties.
If this test becomes invalid there should be a very good reason for invalidating it.

Reply:
This test is incorrect from the perspective of the (current) specification (a tuple cannot hold `int`). However, I found the test useful otherwise so I wanted to keep it with a minimal change.

Reply:
In this case perhaps the spec is wrong. A tuple should be able to hold an int if it is a compile-time constant.

Reply:
I didn't get it. Could you elaborate a bit more (ideally wrt to the [spec](https://staging.p4.org/p4-spec/docs/P4-16-v1.2.4.html#sec-type-nesting)) please?

Reply:
It seems useful to allow int as a type for a tuple element.
If the spec says that this is not possible, perhaps the right thing is to change the spec.